DEVELOPMENT.md

```diff
@@ -199,7 +199,9 @@ Then install Knative Serving/Eventing on it following [the standard
 instructions](https://knative.dev/docs/install/).
 
 ⚠️ **Note:** If you are using a private registry for development you will need
-to grant the ServiceAccount access to your private repository. For GKE you would
+to pre create a ServiceAccount that have the required info and set the `Spec.ServiceAccountName` parameter.
+
+You can also grant the normal ServiceAccount access to your private repository. For GKE you would
 do it like so:
```

Comment:
you will need to ~~pre~~ create a ServiceAccount

`You can also grant the normal ServiceAccount` Not really clear what "normal" service account would be. Should it be default service account?